### PR TITLE
Search results

### PR DESCRIPTION
**Pull request by ZeeshaanMaudar**
Create search and filter options on Homepage.
Search results and refine filter.
Merge mdc-expansion.scss and remove duplicate code.

### CODE REVIEW

File: biz_portal/apps/portal/templates/portal/business_list.html [hunk 1]
```diff
@@ -1,8 +1,89 @@
+{% extends "base.html" %}
 {% load param_replace %}
+{% load static %}
 
-{% include 'portal/search_snippet.html' %}
+<!-- {% include 'portal/search_snippet.html' %} -->
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business_list.html]: we don't keep commented out code. Also django will still insert this snippet even if it's in an HTML comment. If you want to exclude a django tag, use `{# .... #}` I think

File: biz_portal/apps/portal/templates/portal/business_list.html [hunk 2]
```diff
@@ -1,8 +1,89 @@
...
+              </summary>
+            </div>
+            <div class="business-details-status">
+              <summary class="business-summary">
+                <span class="business-details-content">
+                  <span class="status-icon-span">
+                    <img class="status-icon" src="{% static 'images/ellipse.svg' %}" />
+                  </span>
+                  <span class="type-header">Status:</span>
+                  <span class="header-summary">status label here</span>
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business_list.html]: you know how to insert status. Do so. 

et's stop doing placeholders - with this pull request the site is ready to be used.

File: biz_portal/apps/portal/templates/portal/business_list.html [hunk 3]
```diff
@@ -1,8 +1,89 @@
...
+        <div class="business-name mdc-theme--primary">{{ business.registered_name }}</div>
+
+        <hr class="mdc-list-divider">
+
+        <div class="location-and-status-container">
+          <div class="business-details-location">
+              <summary class="business-summary">
+                <span class="business-details-content">
+                  <i class="material-icons mdc-list-item__graphic mdc-theme--primary" aria-hidden="true">location_on</i>
+                  <span class="header-summary">Address here</span>
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business_list.html]: this isn't address, it's region - skip the muni - the portal is for one muni. So just template region in here. You know how.

File: biz_portal/apps/portal/templates/portal/business_list.html [hunk 4]
```diff
@@ -1,8 +1,89 @@
...
+        </span>
+      </div>
+    {% endif %}
+  </div>
+</div>
+
+{% endblock content %}
 
-<h1>Businesses</h1>
 <ul>
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/business_list.html]: why is there markup outside the content block? Should this be deleted? Is this unused? Don't think it'll render if it's outside the content block.

File: biz_portal/apps/portal/templates/portal/search_snippet.html [hunk 5]
```diff
@@ -9,7 +9,14 @@
   <input type="hidden" name="region" value="{{ request.GET.region }}" />
   {% endif %}
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/search_snippet.html]: is this file still used? If not it should be deleted.

File: biz_portal/apps/portal/templates/portal/search_filter_snippet.html [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+              <div class="location-count">{{ sector.count }}</div>
+              <i class="material-icons mdc-list-item__graphic arrow-right" aria-hidden="true">keyboard_arrow_right</i>
+            </div>
+          </a>
+        {% endfor %}
+      </details>
+      {% endif %}
+
+      </div>
+      <a class="link-clear-search" href="/businesses/">
```
Comment (jbothma) [inline biz_portal/apps/portal/templates/portal/search_filter_snippet.html]: This is better written as
```
href="{% url "business_list" %}"
```

File: biz_portal/apps/portal/templates/portal/business_list.html (hunk 1 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business_list.html]: Covered

File: biz_portal/apps/portal/templates/portal/business_list.html (hunk 2 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business_list.html]: Sorted

File: biz_portal/apps/portal/templates/portal/business_list.html (hunk 3 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business_list.html]: Covered

File: biz_portal/apps/portal/templates/portal/business_list.html (hunk 4 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/business_list.html]: Sorted

File: biz_portal/apps/portal/templates/portal/search_snippet.html (hunk 5 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/search_snippet.html]: Nope. removed

File: biz_portal/apps/portal/templates/portal/search_filter_snippet.html (hunk 6 above)
Comment (ZeeshaanMaudar) [reply biz_portal/apps/portal/templates/portal/search_filter_snippet.html]: Done

**Comment by jbothma** [comment]
JS error on business page

bizportal-js-error-on-business-page

**Pull request merged**